region/norfair/east.json

```diff
@@ -3244,7 +3369,56 @@
                   "notable": false,
                   "requires": [
                     "h_canNavigateHeatRooms",
-                    {"heatFrames": 300}
+                    {"or": [
+                      "canTrickyJump",
+                      {"and": [
+                        "ScrewAttack",
+                        "canCarefulJump",
+                        {"heatFrames": 60}
+                      ]},
+                      {"and": [
+                        {"obstaclesCleared": ["A"]},
+                        {"heatFrames": 60}
+                      ]}
+                    ]},
+                    {"heatFrames": 230}
+                  ]
+                },
+                {
+                  "name": "SpaceJump",
+                  "notable": false,
+                  "requires": [
+                    "h_canNavigateHeatRooms",
+                    "SpaceJump",
+                    {"heatFrames": 180}
+                  ]
+                },
+                {
+                  "name": "Kill the Dragons",
+                  "notable": false,
+                  "requires": [
+                    "h_canNavigateHeatRooms",
+                    {"or": [
+                      {"ammo": {"type": "Super", "count": 3}},
```

Comment:
is a PB kill worth including?

Reply:
I didn't see using PBs helped avoid damage once in the middle of the room, where it gets dangerous.
There could be a duplicate of the avoid strat which uses PBs to clear all enemies.